server/src/session.rs

```diff
@@ -34,12 +34,16 @@ impl Session {
         Self {
             path: OwnedObjectPath::try_from(format!("/org/freedesktop/secrets/session/s{index}"))
                 .unwrap(),
-            _aes_key: aes_key,
+            aes_key,
             manager,
         }
     }
 
     pub fn path(&self) -> &OwnedObjectPath {
         &self.path
     }
+
+    pub fn aes_key(&self) -> &Arc<Key> {
+        self.aes_key.as_ref().unwrap()
```

Comment:
This is wrong. A "plain" session would crash here.

Reply:
Yeah, I missed that. Thanks!